southpark-search/tests/distributed/terraform.tf

```diff
@@ -139,7 +32,33 @@ resource "aws_ecs_task_definition" "southpark_task" {
         }
       ],
       "memory": 512,
-      "cpu": 256
+      "cpu": 10
+    },
+    {
+      "name": "encoder",
+      "image": "416454113568.dkr.ecr.us-east-2.amazonaws.com/sp-repo",
+      "essential": true,
+      "portMappings": [
+        {
+          "containerPort": 80,
```

Comment:
this should be 8000. For all port mappings, you don't need the `hostPort` if `containerPort==hostPort`.

Reply:
I think we can't have the same port for both of them

Reply:
OK, can we then have the same container port, but different host ports?

Reply:
I also tried but it says they should match